File: leetcode/Array/Plus One/Plus_One.cc
class Solution {
public:
    vector<int> plusOne(vector<int>& digits) {
        int len = digits.size()-1;
        while(len >= 0){
            if((digits[len] == 9)&&(len == 0)){
                digits[len] = 0;
                digits.insert(digits.begin(), 1);
                
            }else if(digits[len] == 9){
                digits[len] = 0;
            } else {
                digits[len] = digits[len] + 1;
                break;
            }
            len = len - 1;
        }
        return digits;
    }
};

File: leetcode/Array/Plus One/Plus_One.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        length = len(digits)-1
        while (length >=0):
            if((digits[length] == 9)and(length == 0)):
                digits[length] = 0
                digits.insert(0, 1)
            elif (digits[length] == 9):
                digits[length] = 0;
            else:
                digits[length] = digits[length] + 1;
                break;
            
            length = length - 1
        return digits;
    
    
    
     